wand/image.py

```diff
@@ -1994,6 +2019,29 @@ def transparent_color(self, color, alpha, fuzz=0, invert=False):
                                             alpha, fuzz, invert)
         self.raise_exception()
 
+    def compare(self, image, metric='undefined'):
+        """Compares an image to a reconstructed image.
+
+        :param image: The reference image
+        :type image: :class:`wand.image.Image`
+        :param metric: The metric type to use for comparing.
+        :type metric: :class:`basestring`
+        :returns: The difference image, the computed distortion between
+                  the images
+        :rtype: :class:`wand.image.Image`, :class:`numbers.Integral`
```

Comment:
Seems ambiguous between “image or integer” and “a pair of (image, integer).”  I think simply `tuple` would be better.  The type of elements should go to `:returns:` rather than `:rtype:`.